source/org/zfin/gwt/root/dto/RelatedEntityDTO.java

```diff
@@ -14,8 +14,11 @@
 public class RelatedEntityDTO implements IsSerializable, HasLink, Comparable , Serializable {
 
     // data
+    @JsonView(View.AntibodyAliasAPI.class)
```

Comment:
DO we need a new Json view for this? Or could we use one of the more generic ones, like the API one?